ajax/libs/jquery.mosaicflow/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jquery.mosaicflow",
+  "version": "0.2.5",
+  "description": "Pinterest like responsive image or HTML grid for jQuery that doesn’t suck",
+  "homepage": "https://sapegin.github.io/jquery.mosaicflow/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sapegin/jquery.mosaicflow.git"
+  },
+  "filename": "jquery.mosaicflow.min.js",
```

Comment:
It's good to let `filename` between `name` and `version` field.
Because it would helpful to developers to check which is the main file at the first glance.
Thank you.